stories.json

```diff
@@ -33,5 +33,60 @@
     "permalink": "good-times",
     "text": "We have a holistic view of people. Work life, a family life,  includeanything life are not silos. People have one life. Organizations need to consider that. It is less about partying and onsites (even if they mean a lot to me <3), it is more about having a purpose in your daily work (no bullshit jobs) and the `go` to self-organize. It is about no boundaries to thinking and getting the information you need to make decisions (transparency) and decide as an adult person what is best for you. It is about the flexibility to integrate all elements of life into your day.",
     "background": "northernlights"
+  },
+  {
+    "permalink": "long-run",
+    "text": "We sold a few companies. We are having fun. How do we make it stay that way, how do we make this work?",
+    "background": "gradient"
+  },
```

Comment:
I think this is not a story yet.. Probably it was just an idea of a future story 🤷‍♀️

Reply:
Removed it for now.